### PR TITLE
Add `otlploggrpc` exporter skeleton

### PR DESCRIPTION
**Pull request by XSAM**
part of #5056

### CODE REVIEW

**Review by pellared — APPROVED** [review]

**Review by dashpole — APPROVED** [review]

File: exporters/otlp/otlplog/otlploggrpc/config.go [hunk 1]
```diff
@@ -0,0 +1,226 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package otlploggrpc // import "go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploggrpc"
```
Comment (MrAlias) [inline exporters/otlp/otlplog/otlploggrpc/config.go:4]:
```suggestion
package otlploggrpc // import "go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploggrpc"

```

**Review by MrAlias — APPROVED** [review]

**Comment by pellared** [comment]
@MrAlias, https://github.com/open-telemetry/opentelemetry-go/issues/5248#issuecomment-2069988610 is this comment applicable to this PR?

**Comment by MrAlias** [comment]
> @MrAlias, is this [#5248 (comment)](https://github.com/open-telemetry/opentelemetry-go/issues/5248#issuecomment-2069988610) applicable to this PR?

I went back and forth on that. I would prefer to only add that option, but wonder about the question for consistency. If there are existing users using the options from the other signal exporters they may find it frustrating that they cannot setup this exporter the same way.

**Comment by pellared** [comment]
@XSAM, can you please fix the build failures?

**Comment by MrAlias** [comment]
I was hoping we could have resolved the open comment before merging this :confused:

**Pull request merged**